tests/installation/boot_into_snapshot.pm

```diff
@@ -23,10 +23,9 @@ use base "opensusebasetest";
 
 sub run {
     my ($self) = @_;
-    assert_screen 'linux-login', 200;
+
     select_console 'root-console';
     # 1)
-    assert_script_run('touch /etc/NOWRITE;test ! -f /etc/NOWRITE');
```

Comment:
@Zaoliang do you know what are the cases where a write only partition is expected?